src/olympia/blocklist/models.py

```diff
@@ -363,16 +380,16 @@ def _split_guids_fake(cls, guids, v_min, v_max, filter_existing):
 
         # And then any existing block instances
         block_qs = Block.objects.filter(guid__in=guids).values_list(
-            'id', 'guid', 'min_version', 'max_version', 'kinto_id')
+            'id', 'guid', 'min_version', 'max_version', 'kinto_id', named=True)
         existing_blocks = [
             cls.FakeBlock(
-                id=id_,
-                guid=guid,
-                addon=addon_guid_dict.get(guid),
-                min_version=min_version,
-                max_version=max_version,
-                is_imported_from_kinto_regex=kinto_id.startswith('*'))
-            for id_, guid, min_version, max_version, kinto_id in block_qs]
+                id=block.id,
+                guid=block.guid,
+                addon=addon_guid_dict.get(block.guid),
+                min_version=block.min_version,
+                max_version=block.max_version,
+                is_imported_from_kinto_regex=block.kinto_id.startswith('*'))
+            for block in block_qs]
```

Comment:
this is an unconnected refactoring too when I was experimenting with adding new properties to `FakeBlock` but the code is cleaner now so kept it.